source/features/update-pr-from-base-branch.tsx

```diff
@@ -88,8 +94,9 @@ async function addButton(): Promise<void> {
 		return;
 	}
 
-	for (const heading of select.all('.mergeability-details > :not(.js-details-container) .status-heading')) {
-		heading.append(createButton(base, head));
+	for (const meta of select.all('.mergeability-details > :not(.js-details-container) .status-meta')) {
+		meta.classList.add('rgh-has-update-pr-from-master-button');
+		meta.append(createButton(base, head));
```

Comment:
Basically something like this (but optimized to avoid repetition and unnecessary wrappers)
```suggestion
	for (const meta of select.all('.mergeability-details > :not(.js-details-container) .status-meta')) {
		meta.after(
			<span class="status-meta rgh-has-update-pr-from-master-button">
				{createButton(base, head))}
			</span>
		)
        
```